### PR TITLE
Added policy to restrict clusterrole permissions for mutating and validating admission webhooks 

### PR DESCRIPTION
**Pull request by nsagark**
…idating webhoooks.

## Description
This is Kyverno equivalent policy for below.  
https://github.com/bridgecrewio/checkov/blob/main/checkov/kubernetes/checks/resource/k8s/RbacControlWebhooks.py

ClusterRoles that grant write permissions over admission webhook should be minimized to reduce powerful identities in the cluster. Validating admission webhooks can read every object admitted to the cluster, while mutating admission webhooks can read and mutate every object admitted to the cluster. As such, ClusterRoles that grant control over admission webhooks are granting near cluster admin privileges. Minimize such ClusterRoles to limit the number of powerful credentials that if compromised could take over the entire cluster

## Checklist

- [x] I have read the [policy contribution guidelines](https://github.com/kyverno/policies/blob/main/README.md#contribution).
- [x] I have added test manifests and resources covering both positive and negative tests that prove this policy works as intended.
- [x] I have added the artifacthub-pkg.yml file and have verified it is complete and correct.

### CODE REVIEW

File: other/restrict-clusterrole-mutating-validating-admission-webhooks/restrict-clusterrole-mutating-validating-admission-webhooks.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+              operator: AnyIn
+              value:
+              - mutatingwebhookconfigurations
+              - validatingwebhookconfigurations
+            - key: "{{ element.verbs }}"
+              operator: AnyIn
+              value:
+              - create
+              - update
+              - patch
```
Comment (chipzoller) [inline other/restrict-clusterrole-mutating-validating-admission-webhooks/restrict-clusterrole-mutating-validating-admission-webhooks.yaml:43]: You also must account for wildcards here.

**Comment by nsagark** [comment]
Updated the policy to account for wildcards. Please review.

File: other/restrict-clusterrole-mutating-validating-admission-webhooks/restrict-clusterrole-mutating-validating-admission-webhooks.yaml (hunk 1 above)
Comment (chipzoller) [reply other/restrict-clusterrole-mutating-validating-admission-webhooks/restrict-clusterrole-mutating-validating-admission-webhooks.yaml:43]: `resources` still does not account for wildcards.

**Comment by chipzoller** [comment]
DCO is required here.

**Comment by chipzoller** [comment]
Please sign off on your PR.

**Comment by chipzoller** [comment]
Sign off is not complete.

**Comment by chipzoller** [comment]
Converting to draft until all basic requirements are met.

**Comment by chipzoller** [comment]
@nsagark, do you intend to complete this PR or shall we close it?

**Comment by nsagark** [comment]
@chipzoller I will work on these PR's next week.

**Pull request closed without merging**